extensions/amp-story/0.1/amp-story.css

```diff
@@ -331,6 +461,7 @@ amp-story-grid-layer * {
   top: 0 !important;
   padding: 44px 32px 32px;
   overflow: auto;
+  z-index: 30;
```

Comment:
What's this for?

Reply:
I think it's leftover from some technical exploration, but it shouldn't be a magic number if it is necessary